frontend/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -120,15 +123,22 @@ export class VarianceHomePage extends Component {
         issue_date_before: this.params.issue_date_before,
         expiry_date_before: this.params.expiry_date_before,
         expiry_date_after: this.params.expiry_date_after,
+        ...this.params,
       },
```

Comment:
same here, what is being spread? should be defined

Reply:
removed

Reply:
un removed